client/examples/tutorial.rs

```diff
@@ -185,3 +189,55 @@ fn asset_registration_test(config: &Configuration) -> Result<(), Error> {
     // Finish the test successfully
     Ok(())
 }
+
+fn asset_minting_test() {
+    // BEGIN FRAGMENT: mint_asset
+    use std::str::FromStr;
+use iroha_data_model::{
+    IdBox, Value, prelude::{
+        AccountId, AssetId, AssetDefinitionId, MintBox, 
+    }
+ }
+
+    let roses = AssetDefinitionId::from_str("rose#wonderland")
+        .expect("Valid, because the string contains no whitespace, has a single '#' character and is not empty after");
+    let alice: AccountId = "alice@wonderland".parse()
+        .expect("Valid, because the string contains no whitespace, has a single '@' character and is not empty after");
+
+    let _mint_roses = MintBox::new(
+        Value::U32(42),
+        IdBox::AssetId(AssetId::new(roses.clone(), alice.clone())),
+    );
+    // END FRAGMENT
+
+    // Finish the test successfully
+    Ok(())
+}
+
+fn asset_burning_test() -> Result<(), Error> {
+    // BEGIN FRAGMENT: burn_asset
+    use std::str::FromStr;
+    use iroha_data_model::Value;
+    use iroha_data_model::IdBox;
+    use iroha_data_model::prelude::BurnBox;
+    use iroha_data_model::prelude::AssetId;
+    use iroha_data_model::prelude::AssetDefinitionId;
+    use iroha_data_model::prelude::AccountId;
```

Comment:
Same thing about imports

```suggestion
    use iroha_data_model::{
        IdBox, Value, prelude::{
            AccountId, AssetId, AssetDefinitionId, MintBox, 
        }
     }
```